taskfile/node_http.go

```diff
@@ -60,6 +56,14 @@ func (node *HTTPNode) Remote() bool {
 }
 
 func (node *HTTPNode) Read(ctx context.Context) ([]byte, error) {
+	url, err := RemoteExists(ctx, node.logger, node.URL)
+	if err != nil {
+		return nil, err
+	}
+	node.URL = url
+	if errors.Is(ctx.Err(), context.DeadlineExceeded) {
```

Comment:
Shouldn't this one be before the previous if?

I mean you should  be testing this no ?


```suggestion
	if errors.Is(err, context.DeadlineExceeded) {
```

Said otherwise, is there any case where RemoteExist will not return an error when the context is expired?

Reply:
You're right, it should be the previous if but it's a bit more complexe to fix than apply your commit
After checking, the Timeout error does not work on main.
Thanks for reporting this